README.md

```diff
@@ -60,8 +60,7 @@ Visit the [MathType integrations release notes](https://docs.wiris.com/mathtype/
 | [`@wiris/mathtype-generic`](packages/mathtype-generic)                                  | <a href="https://www.npmjs.com/package/@wiris/mathtype-generic"><img src="https://img.shields.io/npm/v/@wiris/mathtype-generic"></a>                                 | MathType Web for a generic HTML textarea.                   |
 | [`@wiris/mathtype-ckeditor4`](packages/mathtype-ckeditor4)                              | <a href="https://www.npmjs.com/package/@wiris/mathtype-ckeditor4"><img src="https://img.shields.io/npm/v/@wiris/mathtype-ckeditor4"></a>                             | MathType Web for CKEditor4 editor.                          |
 | [`@wiris/mathtype-ckeditor5`](packages/mathtype-ckeditor5)                              | <a href="https://www.npmjs.com/package/@wiris/mathtype-ckeditor5"><img src="https://img.shields.io/npm/v/@wiris/mathtype-ckeditor5"></a>                             | MathType Web for CKEditor5 editor.                          |
-| [`@wiris/mathtype-froala`](packages/mathtype-froala)                                    | <a href="https://www.npmjs.com/package/@wiris/mathtype-froala"><img src="https://img.shields.io/npm/v/@wiris/mathtype-froala"></a>                                   | MathType Web for Froala2 editor.                            |
-| [`@wiris/mathtype-tinymce4`](packages/mathtype-tinymce4)                                | <a href="https://www.npmjs.com/package/@wiris/mathtype-tinymce4"><img src="https://img.shields.io/npm/v/@wiris/mathtype-tinymce4"></a>                               | MathType Web for TinyMCE4 editor.                           |
+| [`@wiris/mathtype-froala`](packages/mathtype-froala)                                    | <a href="https://www.npmjs.com/package/@wiris/mathtype-froala"><img src="https://img.shields.io/npm/v/@wiris/mathtype-froala"></a>                                   | MathType Web for Froala2 editor.                            |                          |
```

Comment:
I would change the last column text for `MathType Web for Froala editor` or `MathType Web for Froala 3 or highest version`